tinyiiod.c

```diff
@@ -186,6 +186,16 @@ void tinyiiod_do_close(struct tinyiiod *iiod, const char *device)
 	tinyiiod_write_value(iiod, ret);
 }
 
+int32_t tinyiiod_do_open_instance(struct tinyiiod *iiod)
+{
+	return iiod->ops->open_instance();
```

Comment:
Do we need these function wrappers ?

Reply:
Yes and no.
If we want to remove the wrappers, we will have to expose the "tinyiiod" structure into "tinyiiod.h". In this way, the user of libtinyiiod will have access to the elements of the structure.